### PR TITLE
fix: wallet seed_words command is broken (see issue #4363)

### PR DESCRIPTION
**Pull request by agubarev**
Description
---
https://github.com/tari-project/tari/issues/4363

Motivation and Context
---
Seed word command is broken on GRPC

` thread 'tokio-runtime-worker' panicked at 'called `Option::unwrap()` on a `None` value', applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs:1210:50`

How Has This Been Tested?
---

### CODE REVIEW

File: base_layer/wallet/src/wallet.rs [hunk 1]
```diff
@@ -697,10 +697,10 @@ where
     /// Remove encryption from all the Wallet db backends. If any backends do not have encryption applied then this will
     /// fail
     pub async fn remove_encryption(&mut self) -> Result<(), WalletError> {
-        self.db.remove_encryption().await?;
         self.output_manager_service.remove_encryption().await?;
         self.transaction_service.remove_encryption().await?;
         self.key_manager_service.remove_encryption().await?;
+        self.db.remove_encryption().await?;
```
Comment (sdbondi) [inline base_layer/wallet/src/wallet.rs:703]: Let's rather not sneak this in :)

File: applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs [hunk 2]
```diff
@@ -1205,47 +1206,38 @@ impl wallet_server::Wallet for WalletGrpcServer {
         }
     }
 
+    /// Returns the contents of a seed words file, provided via CLI
     async fn seed_words(&self, _: Request<tari_rpc::Empty>) -> Result<Response<SeedWordsResponse>, Status> {
         let cli = Cli::parse();
```
Comment (sdbondi) [inline applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs:1246]: TODO: Cli::parse should (ideally) not be used in GRPC, rather the ApplicationConfig should be made available to the GRPC server

**Review by sdbondi — APPROVED** [review]
Definitely, an improvement. Panic cases are handled, but as you pointed out, it's risky to delete a file path controlled by user input.

File: base_layer/wallet/src/wallet.rs (hunk 1 above)
Comment (agubarev) [reply base_layer/wallet/src/wallet.rs:703]: Been waiting for another PR for days, you got me :)

File: applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs (hunk 2 above)
Comment (agubarev) [reply applications/tari_console_wallet/src/grpc/wallet_grpc_server.rs:1246]: yes, a good idea

**Pull request merged**